uint8 motor_id
uint8 direction
uint8 use_ramping
uint8 job_id

float32 rpm

uint32 pulses
uint32 ramping_steps
